### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by Eonfge**
And compile with all available cores

### CODE REVIEW

File: com.github.iortcw.iortcw.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ modules:
   buildsystem: simple
   build-commands:
    # For historic reasons, single- and multiplayer are different binaries
-   - cd /run/build/iortcw-source/SP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make copyfiles
-   - cd /run/build/iortcw-source/MP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make copyfiles
+   - cd /run/build/iortcw-source/SP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make -j$(nproc) copyfiles
+   - cd /run/build/iortcw-source/MP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make -j$(nproc) copyfiles
```
Comment (hfiguiere) [inline com.github.iortcw.iortcw.yaml:31]:
```suggestion
   - cd /run/build/iortcw-source/SP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make -j${FLATPAK_BUILDER_N_JOBS} copyfiles
   - cd /run/build/iortcw-source/MP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make -j${FLATPAK_BUILDER_N_JOBS} copyfiles
```

Comment (Eonfge) [reply com.github.iortcw.iortcw.yaml]: Hey thanks, that's a really quick and helpful response.

**Comment by Eonfge** [comment]
What started as a small spelling fix, turned out to be much greater.

**Comment by hfiguiere** [comment]
You removed the `flathub.json` but the launcher explicitly list an x86_64 binary. Which mean the aarch64 build will be broken.

**Pull request merged**